navigation2/package.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>navigation2</name>
+  <version>0.0.1</version>
+  <description>
+    ROS2 Navigation Stack
+  </description>
+  <author>Oregon Robotics Team, Steve Macenski</author>
+  <maintainer email="oregon.robotics.team@intel.com">Oregon Robotics Team</maintainer>
```

Comment:
oregon.roborics.team@intel.com is not a valid email address yet.

Reply:
@SteveMacenski  Please proceed with the PR, address any review feedback, resolve the merge conflicts and we'll get it integrated. Thanks.

Reply:
Yes please.  Thanks.

Reply:
done